process_report/invoices/NERC_total_invoice.py

```diff
@@ -6,6 +6,12 @@
 
 @dataclass
 class NERCTotalInvoice(invoice.Invoice):
+    """
+    This invoice operates on data processed by these Processors:
+    - ValidateBillablePIsProcessor
+    - NewPICreditProcessor
+    """
+
     INCLUDED_INSTITUTIONS = [
         "Harvard University",
```

Comment:
is there any value in having universities here in a separate table we pull from so that they can more easily be added to?